### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by AMzhanghan** [uber-research/UPSNet#136]
when I train the UPSNet, used the config of "upsnet/experiments/upsnet_resnet50_coco_4gpu.yaml", I got the problem as:

`upsnet/../upsnet/models/resnet.py:285: UserWarning: unexpected key "fc.weight" in state_dict
  warnings.warn('unexpected key "{}" in state_dict'.format(name))
upsnet/../upsnet/models/resnet.py:285: UserWarning: unexpected key "fc.bias" in state_dict
  warnings.warn('unexpected key "{}" in state_dict'.format(name))
upsnet/../upsnet/models/resnet.py:299: UserWarning: missing keys in state_dict: "{'resnet_backbone.res2.layers.0.downsample.1.num_batches_tracked', 'resnet_backbone.res4.layers.3.bn1.num_batches_tracked', 'resnet_backbone.conv1.bn1.num_batches_tracked'
.....
.....
python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
Segmentation fault (core dumped)`

How to solve this problem? Thanks!

**Comment by AMzhanghan** [comment]
use_horovod: false ; I was not used the horovod
